examples/ts-react-chat/src/routes/api.mcp-manual.ts

```
@@ -0,0 +1,162 @@
+ * Uses @modelcontextprotocol/server-everything (keyless, via npx) as the MCP
+ * server. Only OPENAI_API_KEY is required — no MCP-specific API keys needed.
+ */
+import { createFileRoute } from '@tanstack/react-router'
+import {
+  chat,
+  chatParamsFromRequestBody,
+  maxIterations,
+  toServerSentEventsResponse,
+} from '@tanstack/ai'
+import { openaiText } from '@tanstack/ai-openai'
+import { createMCPClient, mcpResourceToContentPart, mcpPromptToMessages } from '@tanstack/ai-mcp'
```

ESLint

[error] Member 'mcpPromptToMessages' of the import declaration should be sorted alphabetically.

(sort-imports)